src/main/java/org/openhab/automation/jrule/internal/engine/JRuleEngine.java

```diff
@@ -470,12 +470,11 @@ private void invokeRuleInternal(JRuleExecutionContext context, JRuleEvent event)
                 method.invoke(rule);
             }
         } catch (IllegalAccessException | IllegalArgumentException | SecurityException e) {
-            JRuleLog.error(logger, context.getMethod().getName(), "Error {}", e);
+            JRuleLog.error(logger, context.getMethod().getName(), "Error calling rule method: {}",
+                    ExceptionUtils.getStackTrace(e));
         } catch (InvocationTargetException e) {
```

Comment:
Could we catch this one along with the previous one - or is the slight distinction in the message necessary?

Reply:
Keep them separated because one catch is before the method execution and the other one is inside the method.